features/support/command_helpers.rb

```diff
@@ -16,7 +16,7 @@ def assertions
     WARNINGS = {
       "FC001" => "Use strings in preference to symbols to access node attributes",
       "FC002" => "Avoid string interpolation where not required",
-      "FC003" => "Check whether you are running with chef server before using server-specific features",
+      # FC003 was yanked and is considered reserved, do not reuse it
```

Comment:
yanked might not make sense to international audience. maybe "FC003 has been deprecated and is considered reserved.."